docs/sources/mimir/references/http-api/index.md

```diff
@@ -350,6 +351,23 @@ The flush endpoint also accepts a `wait=true` parameter, which makes the call sy
 
 > **Note**: The returned status code does not reflect the result of flush operation.
 
+### Prepare for Shutdown
+
+```
+GET,POST /ingester/prepare-shutdown
```

Comment:
Let's mention DELETE too and explain what different methods do on this endpoint.